plugins/UsersManager/lang/en.json

```diff
@@ -63,6 +64,11 @@
         "SuperUserAccessManagementMainDescription": "Super users have the highest permissions. They can perform all administrative tasks such as adding new websites to monitor, adding users, changing user permissions, activating and deactivating plugins and even installing new plugins from the Marketplace.",
         "TheLoginScreen": "The login screen",
         "ThereAreCurrentlyNRegisteredUsers": "There are currently %s registered users.",
+        "TokenAuth": "API Authentication Token",
+        "TokenRegenerateConfirm": "Changing the API authentication token will invalidate the users current token. If the user is currently logged in he may be required to login again. Do you really want to continue?",
```

Comment:
I'm not sure. Is it "will invalidate the users current token" => "will invalidate the users' current token"?